src/db/knexfile.ts

```diff
@@ -7,7 +7,7 @@ dotenv.config({
 });
 const config: { [key: string]: Knex.Config } = {
   development: {
-    client: process.env.DB_CLIENT,
+    client: "postgresql",
```

Comment:
@Panas18 
any reason we did not take the env variable and shift it to String?

Reply:
I was getting an error in my path variable, so I shifted to string for debuggint. I have fixed that error now.